IBAnimatableApp/IBAnimatableApp.xcodeproj/project.pbxproj

```diff
@@ -476,7 +483,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
I think both are ok for this one, but if we merged it in master, we need to keep 10.3 for people who didn't download the beta.